### ISSUE TITLE
Composer Network Install is constantly failing with error REQUEST_TIMEOUT

### ISSUE THREAD

**Issue by Johnnyhoboy** [hyperledger/composer#4237]
## Context

Hello, relatively new Hyperledger developer here learning how to deploy a bna file created on composer playground online, and then deploy it on a local fabric environment through AWS EC2 instances. I've been using the starter plan on IBM Cloud but now would prefer to get the same bna file installed and network started on local fabric. 

My problem is that I've followed the steps listed on this [tutorial](https://hyperledger.github.io/composer/latest/tutorials/deploy-to-fabric-single-org), but I'm stuck on step 7 which is the composer network install command. 

I have seen and attempted these solutions: #3974 , #3731 
where I've changed the docker-compose.yml file under hlfv11:
- CORE_CHAINCODE_STARTUPTIMEOUT=2400s
- CORE_CHAINCODE_EXECUTETIMEOUT=2400s
I have modified the values in connection.json to 2400 as well.

I have also tried the solution on the [composer-knowledge-wiki](https://github.com/hyperledger/composer-knowledge-wiki/blob/latest/knowledge.md#installissues) on Error: REQUEST_TIMEOUT where you create a specific npm.yaml file and npmConfig.txt to pass as parameters, however while the "docker-compose -f npm.yaml up -d" command succeeded, I still cannot run the composer network install.

## Expected Behavior

I should be able to install the bna file exported from composer playground online, and then run composer-network-start

## Actual Behavior

ubuntu@ip-10-0-1-170:~/fabric-tools$ composer network install --card PeerAdmin@cfa-perishable --archiveFile cfa-perishable.bna -o npmrcFile=npmConfig.txt
✖ Installing business network. This may take a minute...
Error: Error trying install business network. Error: No valid responses from any peers.
Response from attempted peer comms was an error: Error: REQUEST_TIMEOUT
Command failed

## Steps to Reproduce

1. cd ~/fabric-tools
    ./downloadFabric.sh
    ./startFabric.sh

2. composer network install --card PeerAdmin@cfa-perishable --archiveFile cfa-perishable.bna -o npmrcFile=npmConfig.txt

3. REQUEST_TIMEOUT error after a few minutes, way before the 2400s 

## Your Environment

* Environment name and version: 
export FABRIC_VERSION=hlfv11
node v8.11.1
composer v0.19.12
npm 6.0.1

* Operating System and version (desktop or mobile): Ubuntu 16.04.4 on AWS EC2

**Comment by Johnnyhoboy** [comment]
UPDATE: After not receiving anything helpful from docker logs, I decided to just clean out my environment and install everything again from scratch. I made sure to set export FABRIC_VERSION=hlfv11 after ./startFabric and then ./createPeerAdminCard.sh. Then followed the tutorial again on deploying bna file to single network. I was able to successfully install the bna file, then ran into the common issue with composer-network-start. So I just followed the wiki solution on it and viola it worked! I was able to follow the tutorial to the very end and ping the network.
